### ISSUE TITLE
Remove this package from suggests in all open source packages

### ISSUE THREAD

**Issue by nikolas-burkoff** [insightsengineering/scda.2021#91]
@shajoezhu @cicdguy @donyunardi 

I don't think we need this on CRAN we can use scda.2022 everywhere for examples

 and we get this Note as well which isn't really fixable

![image](https://user-images.githubusercontent.com/15201933/189335972-460f990d-5fdf-4a65-aefc-f08732725423.png)

**Comment by cicdguy** [comment]
Yeah, good idea. I think this package can be deprecated. We can pick this up as an IDR task.

**Comment by shajoezhu** [comment]
Agreed. I don't think we need the old version for the new releases. We need them for examples in old nest releases

**Comment by nikolas-burkoff** [comment]
Note that I think there's some internal work that uses 2021 too (Biomarker catalog @danielinteractive ) I think

**Comment by danielinteractive** [comment]
@nikolas-burkoff thanks but I think we should not be needing the specific 2021 version, we can switch that to 2022 if needed

**Comment by pawelru** [comment]
> @nikolas-burkoff thanks but I think we should not be needing the specific 2021 version, we can switch that to 2022 if needed

I support this. Basically everything should use "latest" endpoint regardless if it is 2021, 2022 or even 2023.

> Note that I think there's some internal work that uses 2021 too

Can we then broaden the scope of this task to remove all occurrences of 2021 - including internal ones?
